cloudinit/config/cc_mounts.py

```diff
@@ -223,42 +223,75 @@ def suggested_swapsize(memsize=None, maxsize=None, fsys=None):
     return size
 
 
+def create_swapfile(fname, size):
+    """Size is in MiB."""
+
+    errmsg = "Failed to create swapfile '%s' of size %dMB via %s: %s"
```

Comment:
This too should likely be a '%s' format modifier instead of '%d' since size is a string.